<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddHeadingDialog</class>
 <widget class="QDialog" name="AddHeadingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add heading - Manuwrite</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="HeadingLevelLayout">
     <item>
      <widget class="QLabel" name="HeadingLevelLabel">
       <property name="text">
        <string>Heading level:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="HeadingLevelCombobox">
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QCheckBox" name="AutonumberCheckbox">
       <property name="text">
        <string>Add section number automatically</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="AutogenIdentifierRadioButton">
       <property name="text">
        <string>Add identifier automatically</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ManualIdentifierRadioButton">
       <property name="text">
        <string>Add identifier manually</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="NoIdentifierRadioButton">
       <property name="text">
        <string>No identifier</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="MainInputsLayout">
     <item>
      <layout class="QVBoxLayout" name="InputsLabelsLayout">
       <item>
        <widget class="QLabel" name="IdentifeirLabel">
         <property name="text">
          <string>Identifeir</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="TextLabel">
         <property name="text">
          <string>Heading text</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="InputsLineEditsLayout">
       <item>
        <widget class="QLineEdit" name="IdentifierLineEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="TextLineEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddHeadingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddHeadingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
